backoff/_typing.py

```diff
@@ -39,6 +39,6 @@ class Details(_Details, total=False):
 _Jitterer = Callable[[float], float]
 _MaybeCallable = Union[T, Callable[[], T]]
 _MaybeLogger = Union[str, logging.Logger, None]
-_MaybeSequence = Union[T, Sequence[T]]
+_MaybeTuple = Union[T, Tuple[T]]
```

Comment:
You need this ellipsis `Tuple[T, ...]` in annotation to be able to set 2 or more exceptions.